pyximport/pyxbuild.py

```diff
@@ -6,9 +6,18 @@
 import os
 import sys
 
-from distutils.errors import DistutilsArgError, DistutilsError, CCompilerError
-from distutils.extension import Extension
-from distutils.util import grok_environment_error
+try:
+    from distutils.errors import DistutilsArgError, DistutilsError, CCompilerError
+    from distutils.extension import Extension
+    from distutils.util import grok_environment_error
+except ImportError:
+    try:
+        from setuptools.errors import DistutilsArgError, DistutilsError, CCompilerError
+        from setuptools.extension import Extension
+        from setuptools._distutils.util import grok_environment_error
```

Comment:
setuptools does not contains `grok_environment_error `

Reply:
We should probably get rid of that usage eventually. Not necessarily now.